src/lib/dom.generated.d.ts

```diff
@@ -2877,14 +2836,13 @@ interface CSSStyleDeclaration {
     content: string;
     counterIncrement: string;
     counterReset: string;
-    cssFloat: string | null;
+    cssFloat: string;
     cssText: string;
     cursor: string;
     direction: string;
     display: string;
     dominantBaseline: string;
     emptyCells: string;
-    enableBackground: string | null;
```

Comment:
As described [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43201#issuecomment-601731848), this is breaking the current `recharts` typings.

So heads-up to the maintainers of `recharts`: @rapmue, @royxue, @ZheyangSong, @richbai90, @caspeco-dan, @pkeuter, @jrsaunde, @crusectrl, @apalugniok, @RobertStigsson, @kousaku-maron, @iflp, @veddermatic, @g07cha, @gndelia, @dimitropoulos, @eliotball, @vkentta, @fcaylus, and @samuelweckstrom.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43201/commits/bce5bbc752f98d07e55e5b29304de04d03a723a5